server/Models/Domain/User.cs

```diff
@@ -15,14 +15,6 @@ public User(String id, string name, string email)
             Email = email;
         }
 
-        public User(String id, string name, string email, List<Booking> booking)
-        {
-            Id = id;
-            Name = name;
-            Email = email;
-            Bookings = booking;
```

Comment:
Do you need List<Booking> in this file when you remove it here?

Reply:
Yes, we need the   public List<Booking> Bookings { get; set; } = new List<Booking>(); cause we need to set it up in Dbcontext